### ISSUE TITLE
Send out an email with instructions

### ISSUE THREAD

**Issue by leouieda** [opengeophysics/2018-agu-oss#4]
We should probably send out an email to all participants with instructions for setting up their computers. We don't want to waste a bunch of time creating accounts and installing Anaconda. 

Hopefully AGU will provide a way to reach out to participants. The workshop system seems a bit weird. They don't show the names of the conveners on the website.

**Comment by krischer** [comment]
Definitely. A year or two ago, anaconda used to rate-limit when too many downloads happen from a single IP - this hit us at a workshop and it caused a great delay.

I guess we should also write a few paragraphs in the readme on how to actually install it.

How do we want to deal with windows? There might not be a bash available - I know the git for windows that github ships comes with a bash; the windows ubuntu subsystem might also be an option. But I'm really no expert on using windows.
